deployment/environment/devenv/chain.go

```diff
@@ -22,13 +22,18 @@ const (
 	EVMChainType = "EVM"
 )
 
+type CribRPCs struct {
+	Internal string
+	External string
+}
+
 // ChainConfig holds the configuration for a with a deployer key which can be used to send transactions to the chain.
 type ChainConfig struct {
 	ChainID     uint64               // chain id as per EIP-155, mainly applicable for EVM chains
 	ChainName   string               // name of the chain populated from chainselector repo
 	ChainType   string               // should denote the chain family. Acceptable values are EVM, COSMOS, SOLANA, STARKNET, APTOS etc
-	WSRPCs      []string             // websocket rpcs to connect to the chain
-	HTTPRPCs    []string             // http rpcs to connect to the chain
+	WSRPCs      []CribRPCs           // websocket rpcs to connect to the chain
+	HTTPRPCs    []CribRPCs           // http rpcs to connect to the chain
```

Comment:
let's create new entries instead of modifying this. We might simply want to connect to testnet chains with this. Internal and external would be confusing in that case